scripts/download

```diff
@@ -20,14 +20,14 @@ rm -rf ${CONTAINERD_DIR}
 mkdir -p ${CHARTS_DIR}
 mkdir -p ${DATA_DIR}
 
+curl --compressed -sfL https://github.com/k3s-io/k3s-root/releases/download/${VERSION_ROOT}/k3s-root-${ARCH}.tar | tar xf -
+
 if [ ${ARCH} = riscv64 ]; then
-  curl --compressed -sfL https://github.com/CARV-ICS-FORTH/k3s-root/releases/download/20230614/k3s-root-riscv64.tar | tar xf -
+  git clone --single-branch --branch=release-1.1-riscv64 --depth=1 https://github.com/CARV-ICS-FORTH/runc ${RUNC_DIR}
```

Comment:
I'm assuming this is the same as https://github.com/brandond/runc/compare/v1.1.6..v1.1.6-k3s1 ? If upstream runc isn't going to pull that in soon, I might fork runc into the k3s-io fork and we can maintain it there until they do accept it.

Reply:
Yes, that sounds as a good idea.

Reply:
The runc developers accepted the change. We are now waiting for the merge.

Reply:
The PR has been accepted and merged into the `release-1.1` branch.

Reply:
I just updated runc to v1.1.8 which supports RISC-V. The if statement is no longer necessary and has been removed.